account.go

```diff
@@ -107,6 +107,16 @@ func (a *account) SignGeneric(ctx context.Context, data []byte, domain []byte) (
 	return sig, nil
 }
 
+// SignGenericMulti signs multiple generic data roots.
+func (a *account) SignGenericMulti(ctx context.Context, data []byte, domain []byte, accounts []e2wtypes.Account) ([]e2types.Signature, error) {
```

Comment:
This should allow multiple `data` roots, as per the comment i.e. `data` should be of type `[][]byte`.